PSGSuite/Private/EncryptionHelpers.ps1

```diff
@@ -0,0 +1,26 @@
+function Invoke-GSDecrypt {
+    param($String)
+    if ($String -is [System.Security.SecureString]) {
+        [System.Runtime.InteropServices.Marshal]::PtrToStringBSTR(
+            [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR(
+                $String
+            )
+        )
+    }
+    elseif ($String -is [ScriptBlock]) {
+        $String.InvokeReturnAsIs()
+    }
+    else {
+        $String
+    }
+}
+
+Function Invoke-GSEncrypt {
+    param($string)
+    if ($string -is [System.String] -and -not [String]::IsNullOrEmpty($String)) {
+        ConvertTo-SecureString -String $string -AsPlainText -Force
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File 'EncryptionHelpers.ps1' uses ConvertTo-SecureString with plaintext. This will expose secure information. Encrypted standard strings should be used instead.](https://app.codacy.com/manual/scrthq/PSGSuite/pullRequest?prid=5105318)